pkg/snmp/snmpparse/config_snmp.go

```diff
@@ -59,6 +61,38 @@ func ParseConfigSnmp(c integration.Config) []SNMPConfig {
 
 	return snmpconfigs
 }
+func ParseConfigSnmpMain() []SNMPConfig {
```

Comment:
```suggestion
func parseConfigSnmpMain() []SNMPConfig {
```

It seems we don't need to export this function since it's only used in this package.